### DISCUSSION TITLE
Adding subcollection to a collection

### DISCUSSION THREAD

**Discussion by Ezius** [ppy/osu#15397] [Ideas]
I think the title is self explanatory
Ty for reading o/

**Comment by bdach** [comment]
why would you need that? the proposal may be self-explanatory, but i'd like to understand the reasoning for it.

**Reply by Ezius** [reply]
I have a collection for burst/stamina maps and i would like to divide them for "with" and "without" DT.
I could also do another collection that is burst/stamina "with DT" and one "without DT" but I think subcollection could be better maybe? I also just like the option to have it

**Reply by peppy** [reply]
out of scope IMO. just make two collections like

header/collection 1
header/collection 2

**Reply by Ezius** [reply]
alright no big deal 
I hope that its not wasting your time that I just threw a random idea out there that I think would be neat to have personally but a bit unnecessary probably

**Reply by Kiko-yu** [reply]
I think its a great idea, especially for tournament mappool organization.

Being able to add multiple tournament mappools from the same tournament under one collcetion would be pretty cool.

**Comment by abraker95** [comment]
fwiw this has been suggested before with several duplicates. I would consider adding this.
https://osu.ppy.sh/community/forums/topics/184436

Also my collection list is [huge](https://i.imgur.com/nAqBc8k.jpeg) and would love some organizational ability.

**Comment by mikumin27** [comment]
I really wish this would be a thing since it would make collections more clean and not overwhelming when you have alot of collections. If it would be collapsible ofc.

Also you could start specifying map types difficulties and things into collection that have alot of different maps for example: farming or favourites or warmup.

**Comment by Ladwick** [comment]
This seems like a good addition, specially considering tournaments, for example, to create a Collection for the tournament mappool and adding subcollections to it like this:

**-Tournament Collection-**
- Qualifiers NM
- Qualifiers HD
- RO32 FM
- GF DT

Something like that.
